stable/node-local-dns/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: node-local-dns
-version: 1.1.5
-appVersion: 1.22.20
+version: 1.2.0
```

Since it's quite a lot changes and some of these might change the default behaviour of the chart, let's go to version 2.

```suggestion
version: 2.0.0
```